### PR TITLE
Clean up capitalization and dynamic requires

### PR DESCRIPTION
**Pull request by ajacksified**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
   },
   "dependencies": {
     "babel-polyfill": "^6.20.0",
+    "change-case": "^3.0.1",
```
Comment (jolewine) [inline package.json:41]: If we are only using constant out of change-case perhaps we should just import constant-case:^2.0.0 https://github.com/blakeembrey/constant-case

File: src/exobot.js [hunk 2]
```diff
@@ -116,25 +118,28 @@ export class Exobot extends Configurable {
   initPlugins(plugins = {}) {
     /* eslint no-param-reassign: 0 */
     const loadedPlugins = Object.keys(plugins).reduce((p, k) => {
-      const [, config] = plugins[k];
+      let [, config] = plugins[k];
```
Comment (jolewine) [inline src/exobot.js]: This could be let [plugin, config] = plugins[k]; now that we are using let instead of const.

**Review by jolewine — COMMENTED** [review]
Haven't had a chance to test but wanted to put these small thoughts down.

File: package.json (hunk 1 above)
Comment (ajacksified) [reply package.json:41]: The unused code will get shaken out by webpack anyways vOv

**Pull request merged**